hbase-server/src/test/java/org/apache/hadoop/hbase/master/cleaner/TestReplicationHFileCleaner.java

```diff
@@ -146,6 +147,7 @@ public void testIsFileDeletable() throws IOException, ReplicationException {
     assertFalse("Cleaner should not allow to delete this file as there is a hfile reference node "
         + "for it in the queue.",
       cleaner.isFileDeletable(fs.getFileStatus(file)));
+    rq.removeHFileRefs(peerId, Arrays.asList("testIsFileDeletableWithNoHFileRefs"));
```

Comment:
Is it possible to do this in tearDown methods?

Reply:
Thanks for reviewing. The HFileRef is created in a specific test, so I think it makes sense to remove it at the end of this test. Alternatively, what I can do is to remove all HFileRefs in the tearDown method (if any). Do you want me to pursue the latter option?

Reply:
A potential problem of putting the removal at the end of the test is that, if the test fail before arriving this line, the HFileRefs will leave there and may cause later tests to fail. I think it will be better to remove all HFileRefs in the tearDown method if any.

Reply:
That makes sense. Pushed a new commit per your suggestion.